app/controllers/incomeSources/add/BusinessAccountingMethodController.scala

```diff
@@ -56,24 +56,28 @@ class BusinessAccountingMethodController @Inject()(val authenticate: Authenticat
                                               (implicit user: MtdItUser[_], backUrl: String, postAction: Call, messages: Messages): Future[Result] = {
     val userActiveBusinesses: List[BusinessDetailsModel] = user.incomeSources.businesses.filterNot(_.isCeased)
 
-    if (userActiveBusinesses.flatMap(_.cashOrAccruals).distinct.size > 1) {
+    if (userActiveBusinesses.flatMap(_.cashOrAccrualsFlag).distinct.size > 1) {
```

Comment:
I am not sure why we use distinct over userActiveBusinesse, pls remove this, as we expecting data to be distinct here anyway

Reply:
cool, make sense, thanks